### PR TITLE
Update Set-CsTeamsCallingPolicy.md

### PR DESCRIPTION
**Pull request by carolynblanding**

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -287,6 +287,8 @@ Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
+> [!NOTE]
```
Comment (tseward) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:290]: Can you change this to **Note:** as the markdown doesn't render in Windows PowerShell?

**Review by dariomws — APPROVED** [review]
In addition to the comment from @tseward, I think the note should be moved above the yaml section.

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks.

**Pull request merged**